src/pages/index.js

```diff
@@ -4,42 +4,145 @@ import FormValidator from "../components/FormValidator.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupDelete from "../components/PopupDelete.js";
 import { data } from "autoprefixer";
 import {
-  initialCards,
   profileAddButton,
   profileEditButton,
+  avatarEditButton,
+  profileSaveButton,
+  cardCreateSaveButton,
+  avatarSaveButton,
   config,
+  newName,
+  newJob,
   formData,
+  cardElement,
 } from "../Utility/Constant.js";
 
-function handleProfileFormSubmit(formData) {
-  userInfo.setUserInfo(formData);
-  profileEditPopup.close();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "266f9e9c-f310-4db4-b30c-94bce5a357d4",
+    "Content-Type": "application/json",
+  },
+});
+
+let section;
+let userInfo;
+
+api.userInfoAndCards();
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    "#cards-template",
+    handleImageClick,
+    handleCardDelete,
+    handleCardLike
+  );
+  return card.generateCard();
 }
+api.getInitialCards().then((data) => {
+  section = new Section({ items: data, renderer: createCard }, ".cards__list");
+  section.renderItems();
+});

```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**